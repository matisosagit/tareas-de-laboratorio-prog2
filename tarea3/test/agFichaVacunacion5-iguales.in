crearAGFichaVacunacion
igualesAGFichaVacunacion 0
insertarVacunaAGFichaVacunacion -1 100
igualesAGFichaVacunacion 0
igualesAGFichaVacunacion 1 -1 100
insertarVacunaAGFichaVacunacion 100 101
insertarVacunaAGFichaVacunacion 100 102
insertarVacunaAGFichaVacunacion 100 103
insertarVacunaAGFichaVacunacion 100 104
igualesAGFichaVacunacion 0
igualesAGFichaVacunacion 4 -1 100 100 101 100 102 100 103
# comparo contra el mismo árbol (insertado en el mismo orden)
igualesAGFichaVacunacion 5 -1 100 100 101 100 102 100 103 100 104
# comparo contra un árbol con mismos 4 hijos pero en diferente orden
igualesAGFichaVacunacion 5 -1 100 100 102 100 101 100 104 100 103
insertarVacunaAGFichaVacunacion 102 200
insertarVacunaAGFichaVacunacion 102 201
insertarVacunaAGFichaVacunacion 102 202
insertarVacunaAGFichaVacunacion 103 300
insertarVacunaAGFichaVacunacion 103 301
insertarVacunaAGFichaVacunacion 103 302
igualesAGFichaVacunacion 0
# comparo contra el mismo árbol (insertado en el mismo orden)
igualesAGFichaVacunacion 11 -1 100 100 101 100 102 100 103 100 104 102 200 102 201 102 202 103 300 103 301 103 302
# comparo contra el mismo árbol de arriba pero con orden diferente
igualesAGFichaVacunacion 11 -1 100 100 101 100 102 100 103 100 104 102 202 102 201 102 200 103 301 103 302 103 300
# comparo contra un árbol con mismos nodos pero hijos de diferentes padres
igualesAGFichaVacunacion 11 -1 100 100 101 100 102 100 103 100 104 102 300 102 201 102 302 103 200 103 301 103 202
liberarAGFichaVacunacion
Fin