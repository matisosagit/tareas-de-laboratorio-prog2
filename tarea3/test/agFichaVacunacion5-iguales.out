1> Se ejecutó 'crearTAGFichaVacunacion' exitosamente.
2> Las fichas de vacunación son iguales.
3> Se ejecutó 'insertarVacunaTAGFichaVacunacion' con codVacunaPadre -1 y codVacuna 100.
4> Las fichas de vacunación NO son iguales.
5> Las fichas de vacunación son iguales.
6> Se ejecutó 'insertarVacunaTAGFichaVacunacion' con codVacunaPadre 100 y codVacuna 101.
7> Se ejecutó 'insertarVacunaTAGFichaVacunacion' con codVacunaPadre 100 y codVacuna 102.
8> Se ejecutó 'insertarVacunaTAGFichaVacunacion' con codVacunaPadre 100 y codVacuna 103.
9> Se ejecutó 'insertarVacunaTAGFichaVacunacion' con codVacunaPadre 100 y codVacuna 104.
10> Las fichas de vacunación NO son iguales.
11> Las fichas de vacunación NO son iguales.
12> #  comparo contra el mismo árbol (insertado en el mismo orden).
13> Las fichas de vacunación son iguales.
14> #  comparo contra un árbol con mismos 4 hijos pero en diferente orden.
15> Las fichas de vacunación son iguales.
16> Se ejecutó 'insertarVacunaTAGFichaVacunacion' con codVacunaPadre 102 y codVacuna 200.
17> Se ejecutó 'insertarVacunaTAGFichaVacunacion' con codVacunaPadre 102 y codVacuna 201.
18> Se ejecutó 'insertarVacunaTAGFichaVacunacion' con codVacunaPadre 102 y codVacuna 202.
19> Se ejecutó 'insertarVacunaTAGFichaVacunacion' con codVacunaPadre 103 y codVacuna 300.
20> Se ejecutó 'insertarVacunaTAGFichaVacunacion' con codVacunaPadre 103 y codVacuna 301.
21> Se ejecutó 'insertarVacunaTAGFichaVacunacion' con codVacunaPadre 103 y codVacuna 302.
22> Las fichas de vacunación NO son iguales.
23> #  comparo contra el mismo árbol (insertado en el mismo orden).
24> Las fichas de vacunación son iguales.
25> #  comparo contra el mismo árbol de arriba pero con orden diferente.
26> Las fichas de vacunación son iguales.
27> #  comparo contra un árbol con mismos nodos pero hijos de diferentes padres.
28> Las fichas de vacunación NO son iguales.
29> Se ejecutó 'liberarTAGFichaVacunacion' exitosamente.
30> Fin.
